R/beast.R

```diff
@@ -350,6 +352,11 @@ add_pseudo_nodelabel <- function(phylo) {
         ##                     treetext)
         ## }
     }
+    # assign blank labels
+    if (any(phylo$node.label == "")) {
+        blanks <- sum(phylo$node.label == "")
+        phylo$node.label[phylo$node.label == ""] <-  paste("X", 1:blanks, sep="")
+    }
```

Comment:
I don't think this method can solve the problem. when the tree contained `X1` of `node label` or the same `node label`.  This method can't also solve the problem.

for example

```
> xx <- '((1[&x="1"]:1,2[&x="2"]:1)5[&x="3"]:1,(3[&x="4"]:1,4[&x="5"]:1)X1[&x="6"]:1)[&x="7"];'
> trda <- read.beast.newick(textConnection(xx))
> trda@data
# A tibble: 7 x 2
      x node
  <dbl> <chr>
1     1 1
2     2 2
3     3 6
4     4 3
5     5 4
6     6 5
7     7 5
```

I think all `node.label` can be replace with  

```
nnode <- phylo$Nnode
phylo$node.label <- paste("X", 1:nnode, sep="")
```

Even `node.label` is not NULL sometimes. because the `node.label` does not effect the following calculation.